### PR TITLE
Basic User Interface branch

### PR DESCRIPTION
**Pull request by ABDELLANI-Youcef**
In this Pull Request, I:
- created components director;
- created  App, Display, ButtonsPanel, Button stateless functional components
- added properties types for Display, Display, ButtonsPanel, Button;
- added Display, ButtonsPanel containing all necessary buttons on it to the page.

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import Display from './Display';
+import Buttonspanel from './ButtonsPanel';
+
+function App() {
+  return (
+    <div className="App">
```
Comment (Forison) [inline src/components/App.js]: * [x] App has children components wrapped in [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax) short syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/))
Stying is not required, at least not yet hence `<div className="App">` has to be change to `<></>` so we don't create extra uneccessary node.
The point is to draw your attention to the concept of fragment in React.

File: src/components/Button.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import PropTypes from 'prop-types';
+
+const Button = props => {
```
Comment (Forison) [inline src/components/Button.js]: **[OPTIONAL]**
`const Button = ({name}) => {} ` will save you from writing line 4

File: src/components/ButtonsPanel.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import Button from './Button';
+
+const ButtonsPanel = () => (
+  <div>
+    <div>
```
Comment (Forison) [inline src/components/ButtonsPanel.js]: **[OPTIONAL]** 
* [x] Why not store the button names in an array and loop over the array and render Button. Then each time you are rendering a button, you pass the button name as the prop to the `Button` component. 
`buttonNames.map(name=> <button name={name} />)`
All you go to do it to get the order of the name right. This gives a much cleaner code.

**Review by Forison — CHANGES_REQUESTED** [review]
## Status::Changes required❌

# 🙋‍♂️ Youcef

#### You have done a good job on this one, almost all the requirements for this project were met, however there are a few issues we have to address to get this project approved.

* [x] Check my comment below👇

#### That is it, kindly make the changes and submit a re-review💪.

#### Happy coding👨‍💻

Addo Forison

**Comment by ABDELLANI-Youcef** [comment]
Thanks very much for the advice. I followed them;

**Review by Torres-ssf — APPROVED** [review]
### Milestone Approved :+1: 
Hi @ABDELLANI-Youcef :hand: 
Great job on this milestone

Happy coding :computer: 
**Sérgio Torres**

**Pull request merged**